source/geometries/TileHamamatsuBlue.cc

```diff
@@ -106,15 +106,15 @@ namespace nexus {
     // SiPMs
     G4LogicalVolume* sipm_logic = sipm_->GetLogicalVolume();
 
-    for (int i=0; i<n_rows_; i++){
-      for (int j=0; j<n_columns_; j++){
-	       G4int copy_no = i*2*n_columns_ + j + 1;
-         G4double x_pos = -tile_x_/2. + offset_x + sipm_dim.x()/2. + j * sipm_pitch_;
-         G4double y_pos = tile_y_/2. - offset_y - sipm_dim.y()/2. - i * sipm_pitch_;
-         G4double z_pos = (tile_z_+epoxy_depth_)/2. - epoxy_depth - sipm_dim.z()/2.;
+     for (int j=0; j<n_rows_; j++){
+       for (int i=0; i<n_columns_; i++){
+         G4int copy_no = (j+1)*10 + i + 1;
+         G4double x_pos = -tile_x_/2. + offset_x + sipm_dim.x()/2. + i * sipm_pitch_;
+         G4double y_pos =  tile_y_/2. - offset_y - sipm_dim.y()/2. - j * sipm_pitch_;
+         G4double z_pos =  tile_z_/2. - quartz_thick_ - lxe_thick_ - sipm_dim.z()/2.;
```

Comment:
The code is not compiling, because the following variables are not defined: `quartz_thick_`  and  `lxe_thick_`. Please, be careful and review the code: I don't think these variables belong to this class...

Reply:
Fixed, sorry about that...